db/test/helper.js

```diff
@@ -271,9 +288,10 @@ exports.testEntityTable = ({
   // it is treated as a collection of modifier functions to run in parallel to check
   // support for concurrent modifications.
   modifications,
-}) => {
+  // customTests can define additional test cases in the usual Mocha style.  The
+  // parameter is true if the tests are run for THIS_VERSION, otherwise for PREV_VERSION.
+}, customTests = (isThisVersion) => {}) => {
```

Comment:
`isThisVersion` looks like it's currently unused.

Reply:
It's used in test cases where customTests is given (#3115).  It's here just to illustrate.